apps/video/js/video.js

```diff
@@ -1289,6 +1325,22 @@ function handleSliderTouchMove(event) {
   dom.player.fastSeek(dom.player.duration * pos);
 }
 
+function handleSliderKeypress(event) {
+  // The standard accessible control for sliders is arrow up/down keys.
+  // Our screenreader synthesizes those events on swipe up/down gestures.
+  // Currently, we only allow screen reader users to adjust sliders with a
+  // constant step size (there is no page up/down equivalent). In the case
+  // of videos, we make sure that the maximum amount of steps for the entire
+  // duration is 20, or 2 second increments if the duration is less then 40
+  // seconds.
+  var step = Math.max(dom.player.duration / 20, 2);
+  if (event.keyCode === event.DOM_VK_DOWN) {
+    dom.player.currentTime -= step;
```

Comment:
This is the same approach we use here: https://github.com/mozilla-b2g/gaia/blob/master/shared/js/media/video_player.js#L488